### ISSUE TITLE
Add "Focus into Activity Bar" action

### ISSUE THREAD

**Issue by myfonj** [microsoft/vscode#108999]
Analogically to _"Focus into Panel"_, _"Focus into Side Bar"_, _"Focus on * View"_ commands it should be possible to move **focus on active item in the Activity Bar** with single command.

Currently -- with default settings -- it is only possible to <kbd>Shift</kbd><kbd>Tab</kbd> there from _Side Bar_, but is comparatively more cumbersome and it respects visual order instead of "focus active item" approach used by aforementioned commands.

Motivation: with increasing versatility of Action Bar being filled with items contributed by extensions it is desirable to navigate quickly among them using keyboard.  Assigning unique keyboard shortcut for every single one of them is an option, but seems like a unnecessarily cumbersome and "non-scaling".

Proposed solution: introduce new `workbench.action.focusActivityBar` _"View: Focus into Activity Bar"_ command focussing active item in Activity Bar.

<details><summary>Fig. "decently crowded Action Bar" in current VSC</summary>

![Twelve items in Action Bar: Explorer, SCM, GitLens (extension), Timeline, Local History (extension), Run, GistPad (extension), Favorites Explorer (extension), Outline, TODOs (extension) and Extensions](https://user-images.githubusercontent.com/1761395/96564443-75d8d780-12c3-11eb-8bf6-5e6fcde3ddbc.png)

</details>

**Comment by isidorn** [comment]
I like this idea. Also makes sense for accessibility. Adding to october to tackle this milestone.

**Comment by myfonj** [comment]
Wow, impressive implementation pace; deep respect!

I've tried today's insider and have few remarks:

#### 1. With **Activity Bar hidden** `focusActivityBar` has no effect.

This differs from _Focus into Side Bar_, which reveals closed Side Bar first (*1).
I think it would make sense if `focusActivityBar` **revealed it's target** as well in this case.

#### 2. `focusActivityBar` **focuses first visible item**.

(It allows subsequent <kbd>tab</kbd> or cursor keys movement among items and <kbd>enter</kbd> or <kbd>Space</kbd> activation).
I see this is probably good and proved UX pattern and I am totally OK with that.
My initial proposal was based on behaviour known from e.g. the tree view in Side Bar: when focus lands in there it lands directly on the last focused visible item, which in most scenarios is the "active" file.
Maybe it would make sense if Activity Bar had it's **own "last focused" item** that'd defaulting to active one.

---

*1: but fails to actually focus it with first invocation; probably an unrelated bug.

**Comment by isidorn** [comment]
1) Makes sense, pushed a commit for that
2) Makes sense, however for that we would have to keep track for last focused item. Also wer have the F6 feature which is an accessible way to traverse the workbench, not sure if that would be surpsising to blind users if the focus is preserved in activity bar. Probably not.

Anyways if you want to tackle 2 a clean PR would be welcome.
